wgpu-core/src/pipeline.rs

```diff
@@ -243,6 +243,8 @@ pub struct RenderPipelineDescriptor<'a> {
     pub multisample: wgt::MultisampleState,
     /// The fragment processing state for this pipeline.
     pub fragment: Option<FragmentState<'a>>,
+    /// If the pipeline will be used with a multiview render pass.
+    pub multiview: bool,
```

Comment:
would this need to know the number of layers?

Reply:
Tested with the mask set to all 1s, resulted in really bad driver crash on AMD so I added the layer count here.